graphstorm-processing/tests/test_converter.py

```diff
@@ -24,7 +24,7 @@
 @pytest.fixture(name="converter")
 def fixture_create_converter() -> GConstructConfigConverter:
     """Creates a new converter object for each test."""
-    yield GConstructConfigConverter()
+    return GConstructConfigConverter()
```

Comment:
Any reason we change here?

Reply:
This matches the return type of the function, `yield GConstructConfigConverter()` returns a `Generator[GConstructConfigConverter]`, `return GConstructConfigConverter()` returns `GConstructConfigConverter` which is the function signature.

In pytest we use `yield` fixtures when we want to include cleanup/teardown code in the fixture _after_ the `yield`, see https://docs.pytest.org/en/6.2.x/fixture.html#teardown-cleanup-aka-fixture-finalization

Here that's not needed.